iron-dropdown.html

```diff
@@ -18,20 +61,198 @@
       Polymer({
         is: 'iron-dropdown',
 
+        behaviors: [
+          Polymer.IronControlState,
+          Polymer.IronA11yKeysBehavior,
+          Polymer.IronOverlayBehavior,
+          Polymer.NeonAnimationRunnerBehavior
+        ],
+
         properties: {
+          /**
+           * The orientation against which to align the dropdown content
+           * horizontally relative to the dropdown trigger.
+           */
+          horizontalAlign: {
+            type: String,
+            value: 'left'
+          },
 
           /**
-           * True if the menu is open.
+           * The orientation against which to align the dropdown content
+           * vertically relative to the dropdown trigger.
            */
-          opened: {
-            type: Boolean,
-            value: false,
-            reflectToAttribute: true
+          verticalAlign: {
+            type: String,
+            value: 'bottom'
+          },
+
+          /**
+           * The element that should be used to position the dropdown when
+           * it is opened.
+           */
+          positionTarget: {
+            type: Object,
+            observer: '_positionTargetChanged'
+          },
+
+          /**
+           * An animation config. If provided, this will be used to animate the
+           * opening of the dropdown.
+           */
+          openAnimationConfig: {
+            type: Object
+          },
+
+          /**
+           * An animation config. If provided, this will be used to animate the
+           * closing of the dropdown>
+           */
+          closeAnimationConfig: {
+            type: Object
+          },
+
+          _positionRectMemo: {
```

Comment:
I thought `memo` was the noun form of `memoize`. Maybe there is a better word? I'm open to suggestions.